src/Pact/Eval.hs

```diff
@@ -324,7 +324,8 @@ eval' (TModule _tm@(MDModule m) bod i) =
         void $ acquireModuleAdminCapability capMName $ return ()
     -- build/install module from defs
     (g,govM) <- loadModule mangledM bod i g0
-    _ <- computeGas (Left (i,"module")) (GPreWrite (WriteModule (_mName m) (_mCode m)))
+    szVer <- ifExecutionFlagSet' FlagDisablePact45 SizeOfV0 SizeOfV1
+    _ <- computeGas (Left (i,"module")) (GPreWrite (WriteModule (_mName m) (_mCode m)) szVer)
```

Comment:
Can we look at some other way to call `computeGas` so that it can test the flag and populate `GPreWrite` as needed? As it is it adds copypasta to every single callsite.

Reply:
I'm fine with making `getSizeOfVersion` because we may fork again later on and match on a different flag.

I don't see a reason to try to be so clever about `computeGas` though. Adding manually 1 argument to like 6 callsites isn't worth abstracting over I think.